.github/workflows/generate.yml

```diff
@@ -54,6 +52,9 @@ jobs:
         repository: googleapis/google-cloudevents
         path: ${{ env.DATA_SOURCE_CHECKOUT_PATH }}
 
+    - name: Set Environment
+      run: echo "GENERATE_DATA_SOURCE=$(realpath ../protos-source-repo)" >> $GITHUB_ENV
```

Comment:
This step is needed because `env` can't be dynamic. I could concatenate variables here... could you tell me more about the preference for variable concat over realpath?

Reply:
Filed follow-up https://github.com/googleapis/google-cloudevents-go/issues/150